shardingsphere-infra/shardingsphere-infra-binder/src/main/java/org/apache/shardingsphere/infra/binder/segment/select/projection/impl/AggregationProjection.java

```diff
@@ -52,7 +51,7 @@
     
     @Override
     public final String getExpression() {
-        return SQLUtil.getExactlyValue(type.name() + innerExpression);
```

Comment:
@iamkyu I found that there are many scenarios where `getExpression` is used for logical judgment. Should these scenarios use `SQLUtil.getExactlyValue(each.getExpression())` to ensure that exact values are used for judgment?

Reply:
@strongduanmu thank you for the review.
I'll check it out. 
Or do you have any better ideas than fix the `AggregationProjection#getExpression()`?

Reply:
Hi @strongduanmu,
I checked where "`getExpression` is used, 
and corrected some scenarios where exact values are needed.